### ISSUE TITLE
Error when using background css property with url

### ISSUE THREAD

**Issue by ffesseler** [halfzebra/create-elm-app#16]
```
.header-container {
  background:url("./images/barre-grise.png") repeat-x 0 0;
}
```

Adding this in main.css throws this error :

```
~/K/l/licence-elm-client (test-elm) $ elm-app start
Running /usr/local/lib/node_modules/create-elm-app/node_modules/.bin/elm-make /Users/toto/licence/licence-elm-client/src/Main.elm --yes --warn --output /var/folders/vj/sl_9sk296rq4
st_49brcbbf80000gn/T/116825-19365-119ploe.js
Success! Compiled 1 module.
Successfully generated /var/folders/vj/sl_9sk296rq4st_49brcbbf80000gn/T/116825-19365-119ploe.js

Compilation failed.

./src/images/barre-grise.png
Module parse failed: /Users/toto/licence/licence-elm-client/src/images/barre-grise.png Unexpected character '�' (1:0)
You may need an appropriate loader to handle this file type.
SyntaxError: Unexpected character '�' (1:0)
    at Parser.pp$4.raise (/usr/local/lib/node_modules/create-elm-app/node_modules/webpack/node_modules/acorn/dist/acorn.js:2221:15)
```

**Comment by joakin** [comment]
This should be fixed with 0c062da right? Next new release should solve this problem.

**Comment by ffesseler** [comment]
Yes, it indeed fixed the background:url issue, thanks !

I've another issue with the img src attribute. I guess the html-loader should be used for that but I don't know much about webpack.
